src/kakarot/accounts/library.cairo

```diff
@@ -235,6 +235,16 @@ namespace AccountContract {
         let (contract_address) = get_contract_address();
         let (balance) = IERC20.balanceOf(native_token_address, contract_address);
 
+        let is_gas_limit_too_high = is_le_felt(2 ** 64, tx.gas_limit);
+        with_attr error_message("Gas limit too high") {
+            assert is_gas_limit_too_high = FALSE;
+        }
+
+        let is_max_fee_per_gas_too_high = is_le_felt(2 ** 128, tx.max_fee_per_gas);
```

Comment:
Couldn't this be replace by `[range_check_ptr] = tx.max_fee_per_gas` ?

Reply:
yes, if you want to verify that tx.max_fee_per_gas is in bound [0, 2**128[

Reply:
I think it would worth changing the condition as it could be cheaper instead of using `is_le_felt`. I will test it.

Reply:
Was cheaper with direct range_check_ptr usage. Change done with [0cad9e5](https://github.com/kkrt-labs/kakarot/pull/1329/commits/0cad9e55d56d223597643b4405b557bdae9a027c)

Before 4048 after 4030 for the test `test_pass_authorized_pre_eip155_transaction`

Before 
![profile001](https://github.com/user-attachments/assets/5c8bf14b-4302-4784-8290-554c00688aba)
After
![profile002](https://github.com/user-attachments/assets/c262abb6-bfb1-4dd6-9456-05333ed72952)